app/js/main.js

```diff
@@ -139,14 +140,17 @@ function closeAnimator() {
 
 function startup() {
   "use strict";
-  // There is no set save directory, set one
-  if (!_getSaveDirectory()) {
+
+  let saveDirectory = _getSaveDirectory();
+
+  // There is no set save directory or the directory does not exist
+  if (!_checkSaveDirectory(saveDirectory)) {
```

Comment:
If this check fails some method of blocking the capturing of frames needs to occur